virtwho/configure.py

```diff
@@ -1,4 +1,119 @@
+                self.update('server', config.xen.server)
+                self.update('username', config.xen.username)
+                self.update('password', config.xen.password)
+            elif mode == 'hyperv':
+                self.update('server', config.hyperv.server)
+                self.update('username', config.hyperv.username)
+                self.update('password', config.hyperv.password)
+            elif mode == 'rhevm':
+                self.update('server', config.rhevm.server)
+                self.update('username', config.rhevm.username)
+                self.update('password', config.rhevm.password)
+            elif mode == 'libvirt':
+                self.update('server', config.libvirt.server)
+                self.update('username', config.libvirt.username)
+                self.update('password', config.libvirt.password)
+            elif mode == 'kubevirt':
+                self.update('kubeconfig', config.mode.config_file)
+            if register == "rhsm":
+                self.update('owner', config.rhsm.default_org)
+            elif register == 'satellite':
+                self.update('owner', config.satellite.default_org)
+            self.update('hypervisor_id', 'hostname')
+
+    def update(self, option, value):
+        """Add or update an option
+        :param option: Option will be added if not exist.
+        :param value: Value to update for the option.
+        """
+        self.cfg.update(self.section, option, value)
+
+    def delete(self, option):
+        """Delete an option
+        :param option: Option to remove.
+        """
+        self.cfg.delete(self.section, option)
+
+    def destroy(self):
+        """Remove the /etc/virt-who.d/{mode}.conf file"""
+        self.remote_ssh.remove_file(self.remote_file)
+
+
+class VirtwhoGlobalConfig:
+    """Able to manage /etc/virt-who.conf file when call this class"""
+    def __init__(self, mode=None):
+        """
+        :param mode: Hypervisor mode. When mode is local, will manage
+            the local libvirt host, othervise will manage the host for
+            all other remote modes.
+        """
+        self.remote_ssh = virtwho_ssh_connect(mode)
+        self.local_file = '/root/virt-who.conf'
+        self.remote_file = '/etc/virt-who.conf'
+        self.cfg = Configure(self.local_file, self.remote_ssh, self.remote_file)
+
+    def update(self, section, option, value):
+        """Add section, add option or update option
+        :param section: Section will be added if not exist.
+        :param option: Option will be added if not exist.
+        :param value: Value to update for the option.
+        """
+        self.cfg.update(section, option, value)
+
+    def delete(self, section, option=None):
+        """Remove a section or option
+        :param section: Section will be removed when no option provided.
+        :param option: Option to remove.
+        """
+        self.cfg.delete(section, option)
+
```

Comment:
How about to add a function to verify the /etc/virt-who.conf in the class `VirtwhoGlobalConfig`?
```
    def verify(self):
        read the virt-who.conf.save file
        check the options of #[global] is 9?
        check the options of #[defaults] is 3?
        check the options of #[system_environment] is 3?
        check each option has the comment?
```
You may add this function by a new pr.

Reply:
Good suggestion and will fix it in another pr.